beta/src/styles/sandpack.css

```
@@ -211,8 +211,14 @@ html.dark .sp-wrapper {
   -webkit-font-smoothing: auto; /* Improve the legibility */
 }
 
+.sandpack .sp-code-editor .cm-content {
+  overflow-x: auto;
```

`auto` keeps the scrollbar visible always. I guess scroll would be better.
```suggestion
      overflow-x: scroll;
```